docs/competition/team-support/battery-charging.md

```diff
@@ -7,44 +7,78 @@ original:
+Every volunteer performing the Battery Charging role must have read this document fully. Any doubts must be raised with the person who briefed you. If you weren't briefed, speak to the volunteer manager.
 
 ## Safety
 
 The biggest risk of damage or injury is from the banana plugs coming unplugged from the charger. These are the red and black plugs that go into the side of the charger. All of the chargers should have the cables cable-tied to the charger to make it difficult to remove these plugs. Under no circumstances should these red and black plugs be unplugged from the chargers.
 
-The batteries themselves have many built in safety features and, unless severely abused, are completely safe. All batteries undergoing charging must be placed inside a fireproof charging bag. In the extremely unlikely event of a fire due to a battery DO NOT attempt to tackle the fire - the batteries contain a flammable metal (Lithium) which cannot be safely tackled with most fire extinguishers.
+The batteries themselves have many built-in safety features and, unless abused, are completely safe. All batteries undergoing charging must be placed inside a fireproof charging bag. In the extremely unlikely event of a fire due to a battery DO NOT attempt to tackle the fire - the batteries contain a flammable metal (Lithium) which cannot be safely tackled by most fire extinguishers.
 
-If at any point you have any doubts about safety, immediately summon the Competition Team Support Coordinator (Rich Barlow) and the Health & Safety Coordinator (Scarzy) via [Helpdesk](/competition/team-support/helpdesk).
+If at any point you have any doubts about safety, immediately summon the Health & Safety officer over the radio.
 
 ## Procedures
 
-### Swapping a team's battery
+### Swapping a team's battery - Helpdesk
+
+Teams will bring discharged batteries to helpdesk and require a replacement charged battery. A Battery Charging volunteer should receive the discharged battery from the team and place it a battery bag in the discharged battery box. A charged battery should be removed from a battery bag in the the charged battery box and handed to the team. Do not allow teams to take batteries themselves - we want to prevent them from hoarding batteries.
+
+### Swapping batteries - Shuffling
 
-Teams will bring discharged batteries to the Battery Charging station and require a replacement charged battery. A Battery Charging volunteer should receive the discharged battery from the team and place it in the discharged battery box. A charged battery should be removed from the charged battery box and handed to the team. Do not allow teams to take batteries from the box themselves - we want to prevent teams from hoarding batteries.
+The volunteer that is running the battery charging station will periodically have to go and get discharged batteries from the helpdesk, simultaneously dropping off charged batteries. Helpdesk may also contact the volunteer over radio to do it sooner.
```

Comment:
Since the station is staffed by at least two people, perhaps phrase this as:
```suggestion
One of the volunteers that is running the battery charging station will periodically have to go and get discharged batteries from the helpdesk, simultaneously dropping off charged batteries. Helpdesk may also contact them over radio to do it sooner.
```